dandi/tests/fixtures.py

```diff
@@ -390,6 +395,87 @@ def zarr_dandiset(new_dandiset: SampleDandiset) -> SampleDandiset:
     return new_dandiset
 
 
+@pytest.fixture()
+def create_video_files(tmp_path):
```

Comment:
Fixtures should have noun names, not verb names, so this should be just "`video_files`."

Reply:
yes makes sense given that they work as variables inside tests

Reply:
https://github.com/dandi/dandi-cli/pull/841/commits/a43c81a7aa990ddb66def6b9f93633c410b6910c